packages/webapp/components/posts/PostWidgets.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ReactElement, useContext } from 'react';
+import dynamic from 'next/dynamic';
+import { PageSidebar } from '@dailydotdev/shared/src/components/utilities';
+import { ShareMobile } from '@dailydotdev/shared/src/components/ShareMobile';
+import AuthContext from '@dailydotdev/shared/src/contexts/AuthContext';
+import { postAnalyticsEvent } from '@dailydotdev/shared/src/lib/feed';
+import AnalyticsContext from '@dailydotdev/shared/src/contexts/AnalyticsContext';
+import { PostData } from '@dailydotdev/shared/src/graphql/posts';
+
+const ShareBar = dynamic(
+  () => import('@dailydotdev/shared/src/components/ShareBar'),
+  {
+    ssr: false,
+  },
+);
+
+const FurtherReading = dynamic(
+  () =>
+    import(
+      /* webpackChunkName: "furtherReading" */ '../widgets/FurtherReading'
+    ),
+);
```

Comment:
This doesn't require dynamic import anymore

Reply:
Good catch! Since this component won't be reached if there's no `post` (as we return a Fragment early on), I have removed the other dynamic imports here as well.